actors/core/src/main/java/cloud/orbit/actors/runtime/AbstractActor.java

```diff
@@ -240,8 +240,6 @@ protected String runtimeIdentity()
      *
      * @return unique identity string
      */
-
-    @Deprecated
```

Comment:
hope you don't mind, we use this all over our project and it seems much easier to access the ref id string directly instead of unwrapping the id through 3-4 method calls.